projects/step-core/src/lib/modules/basics/services/auth.service.ts

```diff
@@ -3,11 +3,12 @@ import { HttpClient } from '@angular/common/http';
 import { Inject, Injectable, OnDestroy } from '@angular/core';
 import { downgradeInjectable, getAngularJSGlobal } from '@angular/upgrade/static';
 import { ConfigDto, CredentialsDto, SessionDto } from '../../../domain';
-import { BehaviorSubject, firstValueFrom, Observable } from 'rxjs';
+import { BehaviorSubject, firstValueFrom, flatMap, Observable, tap } from 'rxjs';
```

Comment:
Pls, clean up the imports that are no longer used - flatMap, switchMap, mergeMap